src/main/java/mate/academy/Main.java

```diff
@@ -1,7 +1,49 @@
 package mate.academy;
 
+import mate.academy.dao.BookDao;
+import mate.academy.lib.Injector;
+import mate.academy.model.Book;
+
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Optional;
+
 public class Main {
+    private static final Injector injector = Injector.getInstance("mate.academy");
+
     public static void main(String[] args) {
+        BookDao bookDao = (BookDao) injector.getInstance(BookDao.class);
+
+        Book book = new Book("Principles of Programming", new BigDecimal("100.00"));
+        System.out.printf("New book created: %s",
+                bookDao.create(book));
+        System.out.println("\n\nLet's try to get our book by ID...");
+        long id = book.getId();
+        Optional<Book> actual = bookDao.findById(id);
+        System.out.printf("Book got by id is: %s", actual.toString());
```

Comment:
Calling `toString()` on an `Optional` without checking if a value is present can lead to misleading output. Consider using `actual.ifPresentOrElse(...)` to handle both the presence and absence of a value more gracefully.